### PR TITLE
update theme

### PR DESCRIPTION
**Pull request by OriolAbril**
I am switching the pydata-sphinx-theme to pymc-sphinx-theme.
It is a very thin wrapper over the pydata-sphinx-theme to abstract the common
logic between the 3 websites, so we don't need to make changes to the 3
repos every time we want to update something or something breaks.

**Checklist**
+ [x] Explain important implementation details 👆
+ [ ] Make sure that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html).
+ [ ] Link relevant issues (preferably in [nice commit messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html))
+ [ ] Fill out the short summary sections 👇

## Docs / Maintenance
- Change theme to pymc-sphinx-theme

### CODE REVIEW

**Review by canyon289 — APPROVED** [review]
Assuming docs build this looks good

**Comment by OriolAbril** [comment]
It should build _and_ 1) look the same + 2) have the same functionality. We have the check the preview very carefully for this PR

**Comment by OriolAbril** [comment]
Only yhe edit this page button seems to not be working. I'll take a look tomorrow

File: docs/source/conf.py [hunk 1]
```diff
@@ -222,47 +226,17 @@ def setup(app):
...
-html_theme = "pydata_sphinx_theme"
+html_theme = "pymc_sphinx_theme"
 
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 
 html_theme_options = {
-    "icon_links": [
```
Comment (OriolAbril) [inline docs/source/conf.py:233]: @twiecki answering https://github.com/pymc-devs/pymc-examples/pull/465#discussion_r1021708138 here. If you look at the preview you will see that the logos are still there. What is being removed is the triplication of this info. Now this is part of the pymc-sphinx-theme which inherits from pydata-sphinx-theme and sets up most of the common elements between al 3 main websittes (and others like pymc-experimental or pymc-bart)

Comment (twiecki) [reply docs/source/conf.py:233]: Amazing!

**Review by twiecki — APPROVED** [review]

**Pull request merged**